lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,34 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.GetAll()
+    local entCount = 0
+    local entCounts = {}
+    for _, ply in pairs( targetPlayers ) do
+        entCounts[ply] = 0
+    end
+
+    for _, ent in pairs( entities ) do
+        local owner = ent:CPPIGetOwner()
+        if entCounts[owner] then
+            local cantFreezeIf = ent:IsWeapon() or ent:GetUnFreezable() or ent:IsPlayer()
```

Comment:
I like canNotFreeze or cantFreeze better or maybe even isUnfreezeable . its fine as this though.